src/validation.py

```diff
@@ -265,6 +265,55 @@ def test_for_missing_energy_source_code(df):
     return missing_esc_test
 
 
+def check_non_missing_cems_co2_values_unchanged(cems_original, cems):
```

Comment:
As we write more validation checks, I think we should separate them into different source files based on either type of validation or where in the pipeline the validation happens. I'd like to avoid getting another 1000+ line file (eg, data_cleaning.py) for two reasons: 1) makes it difficult for to know where to look for code -- you have to either track down a function from data_pipeline.py or know what it's called in order to find it; and 2) when we eventually package OGE, having sensible divisions of logic between files will by default result in useful module names and sensible divisions of logic between modules

Reply:
Good point. I'll add a ticket for this. Maybe we can split this up once we've written more tests so that we can identify what makes sense as a division?